source/more-info/unsupported/cgroup_version.markdown

```diff
@@ -20,5 +20,7 @@ In a supervised installation if you have switched to CGroup v2 you will need to
 revert what you did. Or you can re-run the [supervised installer](https://github.com/home-assistant/supervised-installer)
 to fix it.
 
+On a RaspberryPI 4 add `systemd.unified_cgroup_hierarchy=false` to the file `/boot/cmdline.txt` and reboot afterwards.
```

Comment:
Let's not make this a one-off here, but instead, document it with the Supervised installer itself.

That said, we do consider Home Assistant Supervised to be a highly advanced installation method. These things should not be a problem for the users of such setups.

../Frenck

Reply:
I do have added the information in the supervised installer here: https://github.com/home-assistant/supervised-installer/pull/259

Reply:
Alright, closing the PR on this end 👍